setup.py

```diff
@@ -64,7 +63,6 @@ def read(filename):
         'Development Status :: 4 - Beta',
         'Environment :: X11 Applications :: GTK',
         'License :: OSI Approved :: GNU General Public License (GPL)',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
```

Comment:
Would you mind also adding `python_requires='>= 3.5',` after the classifiers?

Reply:
Sure.